plugins/backend/catalog-backend-module-okta/src/providers/userNamingStrategies/userNamingStrategyFactory.ts

```diff
@@ -32,6 +33,8 @@ export const userNamingStrategyFactory = (
       return kebabCaseEmailUserNamingStrategy;
     case 'strip-domain-email':
       return stripEmailDomainUserNamingStrategy;
+    case 'slugify':
```

Comment:
Should we call this slugify-email, to maintain the reference to the source field in the name, like two strategies?